contracts/EpochTokenLocker.sol

```diff
@@ -63,10 +64,7 @@ contract EpochTokenLocker {
       */
     function deposit(address token, uint256 amount) public {
         updateDepositsBalance(msg.sender, token);
-        require(
-            ERC20(token).transferFrom(msg.sender, address(this), amount),
-            "Tokentransfer for deposit was not successful"
-        );
+        SafeERC20.safeTransferFrom(IERC20(token),msg.sender, address(this), amount);
```

Comment:
Should the type of Token maybe be IERC20 to begin with?

Reply:
Maybe it would be good to do this for basically all addresses that are 'addresses' for erc20 tokens and likewise, in the mappings, which reference to erc20 tokens.

But for coherency, in this PR i would like to stay with the type address